BotDatabase.py

```diff
@@ -80,30 +90,48 @@ def CleanupBackups(self):
                 
     ### Adding/Updating/Removing Server Entries ###
     def AddBotGuilds(self, ListOwnerAndServerTuples, BotID:int):
-        BotAdditionUpdates = []
+        BotAdditionUpdates:list[Server] = []
         for Entry in ListOwnerAndServerTuples:
-            BotAdditionUpdates.append(Entry + (-1, 0, BotID))
+            server = Server(
+                bot_instance_id = BotID,
+                discord_server_id = Entry.id,
+                owner_discord_user_id = Entry.owner_id,
+                activator_discord_user_id = Entry.owner_id
```

Comment:
Could this be 0?

Reply:
as long as value is not null, it can be 0

> With this codebase, `activator_discord_user_id` cannot be null, thus it is set to the owner of the server on first join, and updated by the `interaction.user_id` on state change via `SetBotActivationForOwner`. Ideally, it would initially be set to the person who invited the bot to the server in `on_guild_join`